EIPS/eip-5252.md

```diff
@@ -0,0 +1,187 @@
+---
+eip: 5252
+title: Account-bound Finance
+description: An EIP-5114 extension that aids in preventing loss of funds
+author: Hyungsuk Kang (@hskang9)
+discussions-to: https://ethereum-magicians.org/t/pr-5252-discussion-account-bound-finance/10027
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-06-29
+requires: 721, 1155, 5114
+---
+
+## Abstract
+This EIP proposes a form of smart contract design pattern and a new type of account abstraction on how one's finance should be managed in web3.0, ensuring transparency of managing investments and protection with self-sovereignty even from its financial operators.
+
+This EIP extends [EIP-5114](./eip-5114.md), but the difference is that it can be transferred to other accounts for mobility between managing multiple wallets.
```

Comment:
```suggestion
This EIP extends [EIP-5114](./eip-5114.md) to further enable transferring fund to other accounts for mobility between managing multiple wallets.
```

Reply:
```suggestion
This EIP extends [EIP-5114](./eip-5114.md) to further enable the transfer of funds to other accounts for mobility between managing multiple wallets.
```
I would prefer this wording as it is slightly clearer.